### DISCUSSION TITLE
How to report a bug in pull requests?

### DISCUSSION THREAD

**Discussion by MikeTheGreat** [community/community#23084] [Pull Requests]
<p>Hello!</p>
<p>When opening a pull request I’m not able to see the entire name of the repos that it’s offering to merge from/to.  It’s cut off before the strings become unique, like so:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/5/85f78e780b5baa137b10b1838e948b23c4af98f3.png" data-download-href="/uploads/short-url/j77TeS9Psy87RrT8t2ymGj8Rljd.png?dl=1" title="NameClipped" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/551619/181074745-c99ffec7-b9ef-4c51-8213-342f97b01aac.png" alt="NameClipped" data-base62-sha1="j77TeS9Psy87RrT8t2ymGj8Rljd" width="690" height="302" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f78e780b5baa137b10b1838e948b23c4af98f3_2_690x302.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f78e780b5baa137b10b1838e948b23c4af98f3_2_1035x453.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f78e780b5baa137b10b1838e948b23c4af98f3_2_1380x604.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f78e780b5baa137b10b1838e948b23c4af98f3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">NameClipped</span><span class="informations">2366×1038 267 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>(Org named has been hidden because it’s a work account.)</p>
<p>How do I report this to GitHub as a bug / issue?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s a bug report option in the contact from, bottom row in the middle: <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a></p>

**Reply by Woodz** [reply]
As of 5th Apr 2024, this option is not visible. How do we now report bugs to GitHub?
![image](https://github.com/community/community/assets/966157/4d572a85-3210-4c62-a0c5-f408048979be)

**Reply by esotericist** [reply]
@Woodz it seems to have been relocated down a bit
![image](https://github.com/community/community/assets/1569754/f530d82f-2c6f-4a58-9991-567902dae2c1)

(i'm still grateful for this thread, i had trouble finding the right place to LOOK for the report a bug option, this discussion helped me with that)

**Reply by actualwitch** [reply]
since they keep removing this button from the list (it's not there at all now), here's a link for future reference https://support.github.com/contact/bug-report
